_data/es/settings.yml

```diff
@@ -22,20 +22,20 @@ banner:
 contact_page:
   support_request_form:
     agencies:
-      cbp_jobs: Customs and Border Protection Jobs
-      cbp_roam: Customs and Border Protection ROAM
-      dot_fmcsa: DOT FMCSA (Motor Carrier Safety)
-      dot_nrcme: DOT National Registry of Certified Medical Examiners
-      ds_logon: DS Logon
-      move_mil: DOD Move.mil (Homes or Office)
-      other: Other
-      railroad_retirement_board: Railroad Retirement Board
+      cbp_jobs: Trabajos en Aduanas y Protección de Fronteras
+      cbp_roam: ROAM en Aduanas y Protección de Fronteras  
```

Comment:
More whitespace:

```suggestion
      cbp_roam: ROAM en Aduanas y Protección de Fronteras
```

Separately, we should consider to incorporate something similar to [the YAML normalization](https://github.com/18F/identity-idp/blob/2c2131facd0624c404c247bfa43550ca856e595e/Makefile#L40-L41) in the IDP, to catch these issues more easily.